cli/src/main/java/org/opentosca/toscana/cli/CliProperties.java

```diff
@@ -0,0 +1,99 @@
+package org.opentosca.toscana.cli;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.apache.commons.io.FileUtils;
+
+public class CliProperties {
+
+    private static String filePath;
+    private static String dataPath = null;
+    private static final String dataPathWin = "/AppData/toscana";
+    private static final String dataPathNix = "/.toscana";
+    private static final String cliProp = "/cli.properties";
+    private static Properties properties = new Properties();
+    private static File file;
+    private static String OS = null;
+
+    /**
+     * Creates a cli.properties config file
+     */
+    private static void createProperties() {
+        try {
+            FileWriter writer = new FileWriter(file);
+            properties.setProperty("ip","http://localhost");
+            properties.setProperty("port","8080");
+            properties.setProperty("url","");
+            properties.store(writer, "Cli Settings");
+            writer.close();
+        } catch (IOException e) {
+            System.err.println(e.getMessage());
+        }
+    }
+
+    /**
+     * Gets the API Url from the cli.properties config, if the config doesn't exist a
+     * default config gets created
+     * @return API Url
+     */
+    public static String getApiUrl() {
+        String url = "";
+        setupPath();
+        
+        try {
+            InputStream inputStream = new FileInputStream(file);
+            properties.load(inputStream);
+            inputStream.close();
+        } catch (IOException e) {
+            System.err.println(e.getMessage());
+        }
+        
+        url += properties.getProperty("ip");
+        url += ":" + properties.getProperty("port");
+        url += "/" + properties.getProperty("url");
+        return url;
```

Comment:
url can be an empty string here in case of failure to read properties file.
We should provide a fallback value. maybe use the same value as the default values for the properties file.